lib/java/com/google/android/material/button/MaterialButtonToggleGroup.java

```diff
@@ -249,6 +252,9 @@ public void addView(View child, int index, ViewGroup.LayoutParams params) {
             shapeAppearanceModel.getTopRightCornerSize(),
             shapeAppearanceModel.getBottomRightCornerSize()));
 
+    // Enable children based on the MaterialButtonToggleGroup own isEnabled
+    buttonChild.setEnabled(isEnabled);
```

Comment:
In the [spec](https://lh3.googleusercontent.com/wLirQErMH92A2qj58hHihYhhHG9tNNsOlSqMLjoLb1RwufR7sh0b3mIw-X7DoHOGXT75Rb0FMhCUKsJA7ycfq1qNpeY1NAmYVHXcV_eEcrWi=s0), we want to change individual child button state independently. This will make a previously disabled child button enabled, when the group goes enabled -> disabled -> enabled. But I'm still thinking if we should address it here or as a separate issue. Do you have any thoughts how to make it work like that?

Reply:
I think the MaterialButtonToggle enable or disable enabling all or disabling all is pretty consistent and what one should expect. 
See the issues already open for that (I myself made this PR because i too one day disabled and enabled it expecting this too).
My thought is that Individual child state can still be handled individually. Maybe a comment with a warning should be added along the lines of  "it operates on all childs and if you need a more fine grained aproach please operate on each button individually"
A more cumbersome approach could be added to disable all and on reenable only reenable the ones that where previously enabled but it is even needed? Would that even make sense or be the thing that someone would expect from setEnable(true)?
If you want to explore that (i think it should be other PR) we could explore an overload of the method too.
something like setEnable(bool enable, bool preservePreviousState) but it opens a lot of other considerations and as I think it's complicating a thing that shouldn't be.

Reply:
Yea, I was considering just let this in and deal with that later. This PR still resolves the linked issues. We can take a look at the state restoring later as needed.